src/main/java/org/apache/sling/bundleresource/impl/BundleResource.java

```diff
@@ -186,7 +185,7 @@ Resource getChildResource(final String path) {
         return result;
     }
 
-    private static Object getValue(final JsonValue value, final boolean topLevel) {
+    private static Object getValue(final JsonValue value) {
```

Comment:
same here, seems like a cleanup unrelated to the PR

Reply:
reverted